packages/adobe_analytics/example/android/app/src/main/kotlin/be/vrt/adobe_analytics_example/MainActivity.kt

```diff
@@ -0,0 +1,30 @@
+package be.vrt.adobe_analytics_example
+
+import android.os.Bundle
+import android.util.Log
+import com.adobe.marketing.mobile.Analytics
+import com.adobe.marketing.mobile.Identity
+import com.adobe.marketing.mobile.LoggingMode
+import com.adobe.marketing.mobile.MobileCore
+import io.flutter.app.FlutterActivity
+import io.flutter.plugins.GeneratedPluginRegistrant
+
+class MainActivity : FlutterActivity() {
```

Comment:
Maybe we should do the advised adobe way in our sample app. Which is adding it to a custom Application instead of an Activity

Reply:
Good idea, I've updated the PR 👍